doc/authentication-backends/PKI-authentication-module.md

```diff
@@ -0,0 +1,24 @@
+## Overview
+
+PKI authentication backend is compatible only with SASL mechanisms, that are based on client certificates.
+Currently there is only one such mechanism: `SASL EXTERNAL`.
```

Comment:
Currently only one mechanism meets that requirement and that is`SASL EXTERNAL`?

Currently only `SASL EXTERNAL` meets that requirement?